vaadin-date-time-picker-flow-parent/vaadin-date-time-picker-flow-integration-tests/src/test/java/com/vaadin/flow/component/datetimepicker/validation/DateTimePickerValidationBasicIT.java

```diff
@@ -64,6 +64,17 @@ public void detach_attach_onlyServerCanSetFieldToValid() {
         onlyServerCanSetFieldToValid();
     }
 
+    @Test
+    public void clientSideInvalidStateIsNotPropagatedToServer() {
+        // Make the field invalid
+        $("button").id(REQUIRED_BUTTON).click();
+        testField.sendKeys(Keys.TAB);
```

Comment:
This also needs two tabs apparently:
https://github.com/vaadin/flow-components/blob/cadfe69f0594feeda0389a8b3ea7385de195602b/vaadin-date-time-picker-flow-parent/vaadin-date-time-picker-flow-integration-tests/src/test/java/com/vaadin/flow/component/datetimepicker/validation/DateTimePickerValidationBasicIT.java#L71-L72

Reply:
Right, done.